Makefile

```diff
@@ -7,7 +7,6 @@ BUILD_VERSION ?= $(shell git ls-remote --tags --refs --sort="v:refname" git://gi
 ADVANCED_VET ?= TRUE
 
 LDFLAGS += -X "github.com/mattermost/mmctl/commands.BuildHash=$(BUILD_HASH)"
-LDFLAGS += -X "github.com/mattermost/mmctl/commands.Version=$(BUILD_VERSION)"
```

Comment:
Just a comment, not a proposal. One thing that worries my about having version variables is that they tend to keep outdated.

Reply:
I think that the version variable allows us to keep a different version number for each branch (say, if we have to release a 5.20 and a 5.19.1 versions of the tool), but it is true that we have to be careful for it not to become outdated.